Updated calls to Persona class

apocalyptikz opened the pull request:
Changed Persona.BirthDay to Persona.Birthday

Commit:
Updated calls to Persona class

Changed Persona.BirthDay to Persona.Birthday

Albo1125 reviewed (changes requested):
Thanks for the Persona updates.

For the references etc, please follow the guidelines in the Readme, i.e. use the dependencies folder to include all the references. Also please don't include changes to the csproj file in your commits.

Albo1125 committed:
Delete LSPDFR+.csproj

These changes should not be committed to the public repository to ensure the project keeps working for all contributors, please follow guidelines in the readme.

Albo1125 commented:
Welp, shouldn't have tried reverting the csproj changes on-the-go with the github web interface...

Albo1125 committed:
Revert "Delete LSPDFR+.csproj"

This reverts commit 367a5ed6e7f9f9e0451a06dd3d72ad95d6659b42.

Albo1125 committed:
Reversed changes to the csproj file

Albo1125 commented:
Fixed
